src/ClockMock.php

```diff
@@ -47,7 +47,7 @@ public static function freeze(\DateTimeInterface $dateTime): void
 
         // When freezing, only store $_SERVER['REQUEST_TIME_FLOAT'] the first time (i.e. when the property is null).
         // This way we will restore the actual original value even after freezing multiple times in a row.
-        if (self::$originalServerRequestTimeFloat === null) {
+        if (self::$originalServerRequestTimeFloat === null && isset($_SERVER['REQUEST_TIME_FLOAT'])) {
```

Comment:
Under which circumstances did you find that this is not defined for you? I have tried with the interactive shell (`php -a`) and this is present.